api-specs/api_spec_PGS.yaml

```diff
@@ -787,10 +749,42 @@ components:
           type: string
           description: error description
           example: "Error while requesting authorization for idTransaction: xxx"
-    XPayAuthPollingResponseEntity:
+    XPayPaymentRequest:
+      required:
+        - esito
```

Comment:
to standardize in english the name of this field, like the others

Reply:
XPayPaymentRequest is used as request object to call XPay's Paga3DS API: parameter names in italian are a requirement from XPay